google/samus/pkgs/xf86-input-cmt/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, pkgconfig, xorgserver, xorgproto,
+  utilmacros, libgestures, libevdevc }:
+
+stdenv.mkDerivation rec {
```

Comment:
Cool! I've tried to upstream this way back in https://github.com/NixOS/nixpkgs/pull/29788, but eventually found that libinput works better on my Chromebook device. Have you tried that?

Either way, this should be upstreamed to Nixpkgs. It will bitrot and eventually break here, as an example, see `inversepath/usbarmory` subtree (it doesn't work anymore).

Reply:
I have tried libinput. Don't know if it's just me but the acceleration feels better with cmt. Maybe with tweaking it's possible to get it just right but to me, this feels better.

Yes will do so.